charts/appswitcher-server/templates/basics.yaml

```
@@ -0,0 +1,5 @@
+{{ include "common.serviceAccount" . }}
+
+{{ include "common.service" . }}
+
+{{ include "common.ingress" (dict "ingress" .Values.ingress "service" .Values.service "context" .) }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)